server/config/config.go

```diff
@@ -81,13 +82,13 @@ type buildEventProxy struct {
 }
 
 type DatabaseConfig struct {
-	DataSource             string `yaml:"data_source" usage:"The SQL database to connect to, specified as a connection string."`
-	ReadReplica            string `yaml:"read_replica" usage:"A secondary, read-only SQL database to connect to, specified as a connection string."`
-	StatsPollInterval      string `yaml:"stats_poll_interval" usage:"How often to poll the DB client for connection stats (default: '5s')."`
-	MaxOpenConns           int    `yaml:"max_open_conns" usage:"The maximum number of open connections to maintain to the db"`
-	MaxIdleConns           int    `yaml:"max_idle_conns" usage:"The maximum number of idle connections to maintain to the db"`
-	ConnMaxLifetimeSeconds int    `yaml:"conn_max_lifetime_seconds" usage:"The maximum lifetime of a connection to the db"`
-	LogQueries             bool   `yaml:"log_queries" usage:"If true, log all queries"`
+	DataSource             string        `yaml:"data_source" usage:"The SQL database to connect to, specified as a connection string."`
+	ReadReplica            string        `yaml:"read_replica" usage:"A secondary, read-only SQL database to connect to, specified as a connection string."`
+	StatsPollInterval      time.Duration `yaml:"stats_poll_interval" usage:"How often to poll the DB client for connection stats (default: '5s')."`
```

Comment:
is this safe to change?

Reply:
yup, we just parse the string it into a `Duration` anyway, and both `yaml` and `flag` support `time.Duration` with no changes necessary.